docs/integrations/sources/hubspot.inapp.md

```diff
@@ -1,17 +1,13 @@
-## Prerequisite
+## Prerequisites
 
-* Access to your HubSpot account
+### Authentication method
+You can use OAuth or a Private App to authenticate your HubSpot account.
```

Comment:
we should keep the recommendation for oauth on cloud

Reply:
Roger that